### ISSUE TITLE
Port the code to Plasma 6

### ISSUE THREAD

**Issue by victorhck** [Koffeinfriedhof/gealach#7]
there are plans to port the code to Plasma 6?

greetings

**Comment by vayulove** [comment]
I'm also anxious for a Plasma 6 version.   What I like about this one is the way the lit part fades into darkness.

**Comment by Koffeinfriedhof** [comment]
Due to my state of health in recent years, it is unlikely that I will be able to tackle the project in the near future. Luna III also looks good, can't it simply replace gealach?

**Comment by vayulove** [comment]
> Due to my state of health in recent years, it is unlikely that I will be able to tackle the project in the near future. Luna III also looks good, can't it simply replace gealach?

Of course. The main thing I miss is the fade to dark gealach has, it's subtle, otherwise Luna III is working great.  Thanks for the years that I was able to enjoy gealach.  The value of open source is that the work of one person contributes to all.

**Comment by Koffeinfriedhof** [comment]
Maybe we can just merge the differences. I didn't check out the code, but this seems more likely. Gealach was pretty simple ;) That's something I may be able to do. I'll write it in my Todo-List. Thank you for using it!
